packages/connectivity/connectivity/lib/connectivity.dart

```diff
@@ -63,11 +65,15 @@ class Connectivity {
   ///
   /// From Android 8.0 onwards the GPS must be ON (high accuracy)
   /// in order to be able to obtain the BSSID.
+  @Deprecated(
+      'This method is deprecated. Please use wifi_info_flutter instead. See https://github.com/flutter/plugins/blob/master/packages/wifi_info_flutter/wifi_info_flutter/README.md')
```

Comment:
Why not a link to pub.dev? It'd be much shorter!

Reply:
wifi_info_flutter is not published yet :(